packages/iTwinUI-react/src/core/ComboBox/ComboBoxMenu.tsx

```diff
@@ -56,31 +56,27 @@ const VirtualizedComboBoxMenu = React.forwardRef(
       scrollToIndex: focusedVisibleIndex,
     });
 
-    const overflowY = getWindow()?.CSS?.supports?.('overflow-x: overlay')
-      ? { overflowY: 'overlay' }
-      : { overflowY: 'auto' };
+    const isOverflowOverlaySupported =
+      getWindow()?.CSS?.supports?.('overflow: overlay');
 
-    const styles = React.useMemo(
-      () => ({
-        minWidth,
-        maxWidth: `min(${minWidth * 2}px, 90vw)`,
-      }),
-      [minWidth],
-    );
+    const surfaceStyles = {
```

Comment:
you think component is too light to memoize this object?

Reply:
this object itself is not really expensive to create, so i dont think there's a need to memoize